beavis-ci.sh

```diff
@@ -0,0 +1,205 @@
+#! /usr/bin/env bash
+#=======================================================================
+#+
+# NAME:
+#   beavis-ci.sh
+#
+# PURPOSE:
+#   Enable occasional integration and testing. Like travis-ci but dumber.
+#
+# COMMENTS:
+#   Makes "rendered" versions of all the notebooks listed in the README.rst 
+#   and deploys them to a "rendered" orphan branch, pushed to GitHub for web display.
+#   This script was developed by Phil Marshall as part of the LSST DESC DC2
+#   project: https://github.com/LSSTDESC/DC2_Repo/blob/master/Notebooks/beavis-ci.sh
+#
+# INPUTS:
+#
+# OPTIONAL INPUTS:
+#   -h --help     Print this header
+#   -a --all      Run all the notebooks in the folder instead of the ones in the README
+#   -u --username GITHUB_USERNAME, defaults to the environment variable
+#   -k --key      GITHUB_API_KEY, defaults to the environment variable
+#   -n --no-push  Only run the notebooks, don't deploy the outputs
+#   --html        Make html outputs instead
+#
+# OUTPUTS:
+#
+# EXAMPLES:
+#
+#   ./beavis-ci.sh -u $GITHUB_USERNAME -k $GITHUB_API_KEY
+#
+#-
+# ======================================================================
+
+HELP=0
+no_push=0
+html=0
+all=0
+src="$0"
+
+while [ $# -gt 0 ]; do
+    key="$1"
+    case $key in
+        -h|--help)
+            HELP=1
+            ;;
+        -n|--no-push)
+            no_push=1
+            ;;
+        -a|--all)
+            all=1
+            ;;
+        -u|--username)
+            shift
+            GITHUB_USERNAME="$1"
+            ;;
+        -k|--key)
+            shift
+            GITHUB_API_KEY="$1"
+            ;;
+        --html)
+            html=1
+            ;;
+    esac
+    shift
+done
+
+if [ $HELP -gt 0 ]; then
+  more $src
+  exit 1
+fi
+
+date
+echo "Welcome to beavis-ci: manual occasional integration"
+
+if [ $no_push -eq 0 ]; then
+  if [ -z $GITHUB_USERNAME ] || [ -z $GITHUB_API_KEY ]; then
+      echo "No GITHUB_API_KEY and/or GITHUB_USERNAME set, giving up."
+      exit 1
+  else
+    echo "with deployment via GitHub token $GITHUB_API_KEY and username $GITHUB_USERNAME"
+  fi
+fi
+
+echo "Cloning the DM Stack Club repo into the .beavis workspace:"
+
+# Check out a fresh clone in a temporary hidden folder, over-writing
+# any previous edition:
+\rm -rf .beavis ; mkdir .beavis ; cd .beavis
+git clone https://github.com/LSSTScienceCollaborations/DMStackClub.git
+
+if [ $? -ne 0 ]; then
+  echo "Failed to clone https://github.com/LSSTScienceCollaborations/DMStackClub.git"
+  exit 1
+fi
+
+cd DMStackClub
+
+if [ $html -gt 0 ]; then
+    echo "Making static HTML pages from the master branch notebooks:"
+    outputformat="HTML"
+    ext="html"
+    branch="html"
+else
+    echo "Rendering the master branch notebooks:"
+    outputformat="notebook"
+    ext="nbconvert.ipynb"
+    branch="rendered"
+fi
+mkdir -p log
+webdir="https://github.com/LSSTScienceCollaborations/DMStackClub/tree/${branch}"
```

Comment:
Should this be `StackClub` instead of `DMStackClub`?

Reply:
Oops, sorry: the beavis-ci script should have been removed, since it now has its own repo. See the notes in CIT.md about how to run beavis now: basically you curl it down into your notebooks folder, and then run `beavis-ci.sh LSSTScienceCollaborations/StackClub`.